integreat_cms/static/src/js/push-notifications.ts

```diff
@@ -10,10 +10,20 @@ const switchLanguage = (language: string) => {
     document.getElementById(`li-${language}`).classList.add("z-10", "text-blue-500", "bg-white", "cursor-default");
 };
 
+const adjustTextDirection = (textDirection: string, language: string) => {
+    if (textDirection === "RIGHT_TO_LEFT") {
+        const input = document.getElementById(`tab-${language}`).getElementsByTagName("input")[0] as HTMLElement;
+        const textarea = document.getElementById(`tab-${language}`).getElementsByTagName("textarea")[0] as HTMLElement;
+        input.dir = "rtl";
+        textarea.dir = "rtl";
+    }
+};
+
```

Comment:
> The Typescript code doesn't look create once again, but I didn't know how to improve it. 

I'd say it looks fine as-is. But in case you want to shorten it, here's one option:

```suggestion
const adjustTextDirection = (textDirection: string, language: string) => {
    if (textDirection === "RIGHT_TO_LEFT") {
        (document.querySelector(`#tab-${language} input`) as HTMLElement).dir = "rtl";
        (document.querySelector(`#tab-${language} textarea`) as HTMLElement).dir = "rtl";
    }
};

```